alerts.md

```diff
@@ -81,6 +103,12 @@ notifications:
 				ownerContact: # ownerContact(s) should be the same for the same namespace, otherwise the last namespace alert overwrites
 					- owner@example.com
 					- owner2@example.com
+			- type: spendChange  # change relative to moving avg
```

Comment:
I think we should break these into separate examples so we can explain in more detail.